### ISSUE TITLE
Pulling feels too.... lightweight now?

### ISSUE THREAD

**Issue by Plykiya** [space-wizards/space-station-14#26276]
## Description

Pulling things used to be kind of slow and heavy, now it's so light that everything feels like it's on rollerskates. Rotation of objects as a result of pulling in a certain direction is basically broken as well now, and pulling just feels... way less responsive?

**Reproduction**

**Screenshots**

**Additional context**

**Comment by deepdarkdepths** [comment]
Yeah, I can vouch. Pulling/pushing is incredibly weird and is slightly delayed, it never puts an object where you want it to be

**Comment by superjj18** [comment]
I found out as a borg that if the gravity is off, you can grab a body, drag it forward quickly, and use it to propel yourself rather quickly, and results in a silly effect of the body spinning faster and faster.  Felt like quickly swimming through the station lol.  Not a huge fan

**Comment by KompaniLinge** [comment]
> I found out as a borg that if the gravity is off, you can grab a body, drag it forward quickly, and use it to propel yourself rather quickly, and results in a silly effect of the body spinning faster and faster. Felt like quickly swimming through the station lol. Not a huge fan

this has always kinda been an issue, tbf.

**Comment by KyuPolaris** [comment]
Might be related, but I had an Artifact just tweak out due to a physics bug. Just getting thrown around as if someone was carrying it and ctrl+right clicking

**Comment by Moomoobeef** [comment]
I hope this gets fixed soon because having the mop bucket spill any time you push it is one of the most annoying things I've ever experienced in this game ever.

**Comment by MACMAN2003** [comment]
it also kinda feels like whatever you're dragging overshoots where you want it to go

**Comment by liltenhead** [comment]
You can easily toss people into space, and even toss things like the gravity gen into space with no friction now. They're essentially weightless.

**Comment by Djungelskog2** [comment]
I agree, Id still like it on some smaller furniture (Assuming it doesnt overshoot) but its kinda fucked me over alot like being thrown out of evac pods and failing to position furniture

**Comment by Hanzdegloker** [comment]
The fact you can move full speed while pulling something by just pushing it in front of you is absurd, I've seen people just running thru halls full sprint with bodies occasionally slamming em into walls. Once a borg took my patient off my roller bed and started abusing that fact to get em to medbay faster, I was the paramedic, yes I am still malding I want to feel like a paramedic and use the bloody rollerbed.

**Comment by lzk228** [comment]
closing this, was reverted and "fixed" in https://github.com/space-wizards/space-station-14/pull/28126
